server/models/Column.js

```diff
@@ -0,0 +1,38 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+const Card = require('./Card');
+
+const ColumnSchema = new Schema({
+	name: {
+		type: String,
+		required: true
+	},
+	cards: [Card.schema]
+}, {collection: 'Column'});
+
+// creates a new instance of the Column model and returns the saved instance
+ColumnSchema.statics.createNewColumn = async function(name) {
+	const column = new this({ name });
+	await column.save();
+	return column;
+};
+
+// deletes a Column in the db by its _id
+ColumnSchema.statics.deleteColumn = function(columnId) {
+	this.deleteOne({ _id: mongoose.Types.ObjectId(columnId)}).exec();
+};
+
+// adds a Card model into the Column's cards array
+ColumnSchema.methods.addNewCard = function(card) {
```

Comment:
Might be a good idea to also add a method to Board for addNewCard that takes a card data object and column id and calls this function to create the card and add it to the column